tutorial/4_hextof_workflow.ipynb

```diff
@@ -84,6 +98,8 @@
    },
    "outputs": [],
    "source": [
+    "# file to store the calibration parameters in\n",
+    "cal_file = \"hextof_tutorial_cal.yaml\" \n",
```

Comment:
That is true. Nonetheless, I'd like to make it obvious that the user has the option to change the file where they load the calibration parameters from, and that it doesn't have to be sed_config.yaml. Because I had this question before.

Perhaps instead of here, we put it in the documentation then.